pages/[type]/[id].vue

```diff
@@ -1070,6 +1133,23 @@ function openModerationModal(status) {
   modalModeration.value.show()
 }
 
+async function updateMembers() {
+  allMembers.value = await useAsyncData(
+    `project/${route.params.id}/members`,
+    () => useBaseFetch(`project/${route.params.id}/members`, data.$defaultHeaders()),
+    {
+      transform: (members) => {
+        members.forEach((it, index) => {
+          members[index].avatar_url = it.user.avatar_url
+          members[index].name = it.user.username
+        })
+
+        return members
+      },
+    }
+  )
+}
```

Comment:
why is this needed? when would you update members outside of the members management page?

Reply:
this is called when members accept invites